RadialMotionStateMachine:
    NavigateToRadialStart:
        NavigationOrthogonalLine:
            Navigate:
                start_position_x: 3
                start_position_y: 0
    RotateDegress:
        NavigationOrthogonalLine:
            Navigate:
                initial_orientation_index: 0 # the initial index of the linear motion (factor of angle_increment_degrees)
                angle_increment_degree: 90    # the increment of angle between to linear motions
                linear_trajectories_count: 2  # the number of linear trajectories of the radial motion
    NavigateToEndPoint:
        NavigationOrthogonalLine:
            Navigate:
                straight_motion_distance: 3.5
#    ReturnToRadialStart:
#        NavigationOrthogonalLine:
#            Navigate:
#                sample_param: 1
#        ReelOrthogonalLine:
#            ReelRetract:
#                sample_param: 1
#        ToolOrthogonalLine:
#            ToolSubstate:
#                 sample_param: 1